docs/en/docs/release-notes.md

```diff
@@ -5342,7 +5342,7 @@ Note: all the previous parameters are still there, so it's still possible to dec
     * New documentation about [dependencies in the `include_router()` method](https://fastapi.tiangolo.com/tutorial/bigger-applications/#include-an-apirouter-with-a-prefix-tags-responses-and-dependencies).
     * PR [#235](https://github.com/tiangolo/fastapi/pull/235).
 
-* Fix OpenAPI documentation of Starlette URL convertors. Specially useful when using `path` convertors, to take a whole path as a parameter, like `/some/url/{p:path}`. PR [#234](https://github.com/tiangolo/fastapi/pull/234) by [@euri10](https://github.com/euri10).
+* Fix OpenAPI documentation of Starlette URL converters. Specially useful when using `path` converters, to take a whole path as a parameter, like `/some/url/{p:path}`. PR [#234](https://github.com/tiangolo/fastapi/pull/234) by [@euri10](https://github.com/euri10).
```

Comment:
Here, we should stick to the spelling of `convertors`, as this is how Starlette spells it and how it's used in code: 
```
from starlette.convertors import Convertor, register_url_convertor
```